test/functional/search_controller_test.rb

```diff
@@ -18,6 +18,12 @@ class SearchControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test "search profiles page at /search/profiles/steff?tag=awesome" do
+    get :profiles, params: { query: 'steff', tag: 'awesome' }
```

Comment:
So here, instead of a separate tag, we'd make the query `awesome` and look to see if it returns `steff` because `steff` has the `awesome` tag. 

@milaaraujo please correct me if I'm wrong about this.

Reply:
@jywarren tried just tried this and it did not return `steff` I think its because it tried to search for `awesome` using username and bio.